pkg/api/v1alpha1/pluginapi.proto

```diff
@@ -52,6 +52,8 @@ message ValidateQuoteRequest {
     bytes publicKey = 2;
     // base64 encoded SGX Quote
     bytes quote = 3;
+    // base64 encoded nonce used for generating he SGX quote
```

Comment:
`he` -> `the`. :-) But more seriously, it feels that `ValidateQuoteRequest` and `GetCAkeyAndCertificateRequest` could maybe share a common subtype, because the fields are exactly the same.

Reply:
Yes, I remember you pointed out the same in our initial design of this plugin API. At that time I thought they might differ in the future based on the use case. But I am also leaning towards handling them in a single type. Shall we do it part of this change? or in a separate PR/API version?

Reply:
The change would not be backwards compatible so let's not do it here. But when the api version changes let's revisit.

Reply:
> `he` -> `the`. :-)

Fixed.